packages/composites/synth-index/src/index.ts

```diff
@@ -1,5 +1,7 @@
 import { expose } from '@chainlink/ea-bootstrap'
 import { makeExecute } from './adapter'
+import { NAME } from './config'
 
-export const NAME = 'SYNTH-INDEX'
-export const server = expose(NAME, makeExecute()).server
+const adapterContext = { name: NAME }
+
+export const server = expose(adapterContext, makeExecute()).server
```

Comment:
Since we are changing this line, could we modify this to be standard with a standard export style?
```suggestion
const { server } = expose(adapterContext, makeExecute())
export { NAME, makeConfig, makeExecute, server }
```